### ISSUE TITLE
How to make multiple blogs

### ISSUE THREAD

**Issue by slashtechno** [GetPublii/Publii#1006]
Is it possible to have multiple blogs on one site?

**Comment by internettips** [comment]
I can visualise one way that might work well. Though I think you might need to go to get help: https://forum.getpublii.com

**Comment by iamadlx** [comment]
Yes, depending on where you are hosting, you can have several blogs with several sub-directories
